platform/jewel/ide-laf-bridge/src/main/kotlin/org/jetbrains/jewel/bridge/theme/IntUiBridgeScrollbar.kt

```diff
@@ -33,6 +33,25 @@ internal fun readScrollbarStyle(isDark: Boolean): ScrollbarStyle =
         scrollbarVisibility = readScrollbarVisibility(),
     )
 
+internal fun readTabStripScrollbarStyle(isDark: Boolean): ScrollbarStyle =
+    ScrollbarStyle(
+        colors = readScrollbarColors(isDark),
+        metrics = readScrollbarMetrics(),
+        trackClickBehavior = TrackClickBehavior.JumpToSpot,
+        scrollbarVisibility =
+            ScrollbarVisibility.WhenContainerHovered(
+                trackThickness = 5.dp,
+                trackThicknessExpanded = 5.dp,
+                trackPadding = PaddingValues(1.dp),
```

Comment:
Including on the sides?

Reply:
Yes, the value used in swing is `JBUI.insets(1)` (which is all sides)